packages/main/src/ListItemBase.js

```diff
@@ -12,6 +12,18 @@ import styles from "./generated/themes/ListItemBase.css.js";
 const metadata = {
 	properties: /** @lends  sap.ui.webcomponents.main.ListItemBase.prototype */  {
 
+		/**
+		 * Defines the aria-label attribute of the list item.
+		 * @public
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @since 1.0.0-rc.7
+		 */
+		itemLabel: {
```

Comment:
We should check if we can add aria attrs on the component, when it works, when not, because I have seen libraries to set aria* on the components not in the shadowDOM.

Reply:
I have tried out setting it on the component, but it doesn't work because the focus is on the native ```li``` element.

About the name, I was wondering as well, but since we can't use aria-label, because it is DOM native attribute this was the best I could think of. Any suggestions are welcome

Reply:
Maybe we should think of making the component focusable I have checked the vaadin list component, they seem to se the tabindex (internally) on items level and perhaps it`s working .

<vaadin-list-box role="list" aria-orientation="vertical" >
          <vaadin-item aria-selected="true" tabindex="0" selected="">5k</vaadin-item>
          <vaadin-item disabled="" aria-disabled="true" aria-selected="false" tabindex="-1">10k (sold out)</vaadin-item>
          <vaadin-item aria-selected="false" tabindex="-1">Half marathon</vaadin-item>
          <vaadin-item aria-selected="false" tabindex="-1">Marathon</vaadin-item>
        </vaadin-list-box>

I just want to make sure we have the best approach, because otherwise we will add a lot of properties to the API of the components.

Reply:
You definitely have a point. I will work in that direction